manifests/base/namespaced/leader_election_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: terraform-applier-leader-election
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: terraform-applier-leader-election
+subjects:
+  - kind: ServiceAccount
+    name: terraform-applier
+    namespace: terraform-applier
```

Comment:
I think its better to omit "namespace" here, it will default to "current", which is less surprising behaviour.